### ISSUE TITLE
Getting unusual result from given starting coordinates for last configuration

### ISSUE THREAD

**Issue by WildJimmy** [ubsuny/23-Homework7G1#61]
When using the initial positions provided by the professor for structure 7, I end up with a plot like this

![image](https://github.com/ubsuny/23-Homework7G4/assets/94491866/3f2ce76c-d171-488b-a520-8e125f5e4d99)

Has anyone else had this issue? This is not the form of in the paper and it needs some additional tuning correct?

**Comment by pratibha77118** [comment]
@WildJimmy have you done the seventh configuration ? I have obtained different result then yours. I guess it depends on the initial position coordinates we have chosen.

**Comment by WildJimmy** [comment]
Yeah, this was from the professor's initial positions (though I assume I messed something up somewhere). I ended up changing from 7 since it was proving to be quite annoying

**Comment by AhmedCode99** [comment]
This is interesting and somewhat useful
